src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTaboolaOasis.java

```diff
@@ -19,12 +19,12 @@ public void adsTaboolaOasis() {
     AdsTaboolaObject adsTaboolaObject = new AdsTaboolaObject(driver);
     adsTaboolaObject.getUrl(testedPage);
 
-    adsTaboolaObject.verifyTaboolaContainer(AdsTaboolaObject.RIGHT_RAIL_CSS_SELECTOR);
     adsTaboolaObject.verifyTaboolaContainer(AdsTaboolaObject.BELOW_ARTICLE_CSS_SELECTOR);
-
-    adsTaboolaObject.verifyTaboolaAdsPresent(AdsTaboolaObject.RIGHT_RAIL_CSS_SELECTOR);
     adsTaboolaObject.verifyTaboolaAdsPresent(AdsTaboolaObject.BELOW_ARTICLE_CSS_SELECTOR);
 
+//    @TODO Assertions based on configuration: ADEN-2729
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)